components/colorwheel/index.css

```diff
@@ -56,35 +95,69 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
-}
-
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
+.spectrum-ColorWheel-internal{
```

Comment:
nit: space.

I am debating if we should call the inner area "internal". 
Spectrum React sets up the divs like this and call it "inner".

```
.spectrum-ColorWheel {
  .spectrum-ColorWheel-gradient {
    .spectrum-ColorHandle {
      .spectrum-ColorLoupe {
         .spectrum-ColorLoupe-inner-background {}
         .spectrum-ColorLoupe-inner-checker {}

```

reference: https://react-spectrum.adobe.com/react-spectrum/ColorWheel.html

Reply:
I also like inner better, changed and addressed the spacing inconsistencies.